### DISCUSSION TITLE
[UNSTABLE] EliminateTargets mission type in missions mode can start a ClearAlienRuins mission type

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#13994] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Selecting the EliminateTargets mission type in missions mode and starting a level can start a clearruins mission, even though they are different mission types (only thalamus missions have the EliminateTargets type).

### Reproduction steps

1. Join/Host a multiplayer server.
2. Select Missions mode
3. Disable every mission type EXCEPT EliminateTargets
4. Start a round
4.1. Repeat until wrong mission is selected

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This is not a bug: these are all "Search and Destroy" missions (called EliminateTargetsMissions in the code).
